translator/translate/metrics/global_dimensions/ruleGlobalDimensions.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package global_dimensions
+
+import (
+	parent "github.com/aws/amazon-cloudwatch-agent/translator/translate/metrics"
+)
+
+type globalDimensions struct {
+}
+
+const SectionKey = "global_dimensions"
+
+func (ad *globalDimensions) ApplyRule(input interface{}) (returnKey string, returnVal interface{}) {
```

Comment:
Would appreciate a unit test. It looks straightforward enough but it wouldn't hurt 😄

Reply:
I've added a test for this now.